### ISSUE TITLE
Android apps blurred on high res displays

### ISSUE THREAD

**Issue by NN708** [openFyde/overlay-rock5b-openfyde#7]
After I changed `Display size` in settings, all Android apps are scaled and blurred. Most text in Android apps is large enough, so is there a way to stop this scaling behavior?

**Comment by NN708** [comment]
Thanks for your comment. It is still the same with another account.

**Comment by DarkevilPT** [comment]
> Thanks for your comment. It is still the same with another account.

can you make a video and post it here: https://streamable.com/
I havent actually tried to place the ROCK 5B on my LG TV but I'm curious if it happens to me too.
So far I'm good but this is a low DPI display that I am currently using.

**Comment by NN708** [comment]
@DarkevilPT You can also try that on a low DPI display, just adjust the `Display size` setting.

**Comment by DarkevilPT** [comment]
> @DarkevilPT You can also try that on a low DPI display, just adjust the `Display size` setting.

I did and ... I dont have blurr at all.

**Comment by NN708** [comment]
@DarkevilPT This is what I got under display size 200%:

![Screenshot](https://user-images.githubusercontent.com/20534082/225801061-c3f56ea9-b14f-4a50-a724-d512ad7432a3.png)

**Comment by DarkevilPT** [comment]
https://i.ibb.co/FJF4KBK/Screenshot-2023-03-17-08-12-24.png
I don't think I have that issue tbh... and even your image doesn't look too blurry but yes could be sharper.

**Comment by NN708** [comment]
> https://i.ibb.co/FJF4KBK/Screenshot-2023-03-17-08-12-24.png
> I don't think I have that issue tbh... and even your image doesn't look too blurry but yes could be sharper.

@DarkevilPT It seems that what you are using are all web apps, not Android apps?

**Comment by Damenly** [comment]
Yes. Android apps have DPI scale problems. We are solving those issues.

**Comment by DarkevilPT** [comment]
Did a wee test and yes its somewhat noticeable probably worst on bigger screens which is not my case.
https://streamable.com/klbj83
https://streamable.com/no8dyp

The video recording really puts some bad stuttering to its performance.

**Comment by 333337lee** [comment]
How do you install APK’s?  I can’t find any reference to Android in the settings.

**Comment by DarkevilPT** [comment]
> How do you install APK’s? I can’t find any reference to Android in the settings.

Just download the apk from a web page and install it.

**Comment by 333337lee** [comment]
> How do you install APK’s? I can’t find any reference to Android in the settings.



> 

I tried double clicking or right clicking the APK in Openfyde 108 on a Pi 4. It’s not recognised

**Comment by DarkevilPT** [comment]
> > How do you install APK’s? I can’t find any reference to Android in the settings.
> 
> > 
> 
> I tried double clicking or right clicking the APK in Openfyde 108 on a Pi 4. It’s not recognised

On pi 4? This is the github for the Rock 5B.
Maybe pi 4 hasnt got Arc Hero yet. Check on the pi 4 overlay if it mentions anything about Android or Arch Hero.

**Comment by 333337lee** [comment]
Thank you

I don’t see it mentioned. I had assumed it was a feature in 108. I have now downloaded 108 for Orange Pi 5 but it’s an upgrade not a clean install and the instructions don’t work for me. I filed an issue in the Orange Pi 5 github. 👍🏼

**Comment by DarkevilPT** [comment]
> Thank you
> 
> I don’t see it mentioned. I had assumed it was a feature in 108. I have now downloaded 108 for Orange Pi 5 but it’s an upgrade not a clean install and the instructions don’t work for me. I filed an issue in the Orange Pi 5 github. 👍🏼

But do u have an orange pi 5 ?

**Comment by 333337lee** [comment]
> > Thank you
> > I don’t see it mentioned. I had assumed it was a feature in 108. I have now downloaded 108 for Orange Pi 5 but it’s an upgrade not a clean install and the instructions don’t work for me. I filed an issue in the Orange Pi 5 github. 👍🏼
> 
> But do u have an orange pi 5 ?

Yes, I listed an issue. a new release in a few days 

https://github.com/openFyde/overlay-orangepi5-openfyde/issues/15#issuecomment-1522633114
